Bug 1323536 - Make text field for annotating jobs mandatory for "fixed by commit"

alisha17 opened the pull request:
Frontend change done where minimum length string added

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/treeherder/2062)
<!-- Reviewable:end -->

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

KWierso commented on ui/partials/main/thPinboardPanel.html:
```diff
@@ -50,7 +50,7 @@
       <div class="classification-comment-container">
         <input id="classification-comment"
-               type="text"
+               type="text" 
```
Can you get rid of the trailing whitespace here?

alisha17 replied:
Done !

KWierso commented:
Only other thing I could ask for would be for you to rebase and squash all six commits down into a single commit and then `git push -f` the rebased commit back to your PR branch. I'll test this out more thoroughly tomorrow and then land this. 

Thanks a lot! :)

jmaher commented:
@alisha17 please let us know if you are stuck on squashing and rebasing.

wlach commented:
@KWierso Can we not just use github's squash & merge feature here to do the rebasing on our end?

alisha17 commented:
Although the code is now merged, so it doesn't matters much. But I am curious. I squashed all commits into a single commit. Here: https://github.com/alisha17/treeherder/commit/f51c3c0ad2762e8f7ef6882049166c3a177eb98f .
Isn't this the right procedure for squashing and rebasing?  I first rebased, squashed all commits into one then forced the commit into PR.

edmorley commented:
Your squash looks fine, I think they must have missed the PR update notification email and not rechecked it manually.
